elements/rh-alert/README.md

```diff
@@ -78,3 +78,15 @@ import '@rhds/elements/rh-alert/rh-alert.js';
     <button slot="actions" data-action="confirm">Confirm</button>
 </rh-alert>
 ```
+
+### Adding an Event Listener to a Basic Alert
+
+If you would like to add custom logic when the alert is closed, you can do so with JavaScript. 
+```js
+  // Query for the alert element
+  const alert = document.querySelector('rh-alert');
+  alert.addEventListener('close', () => {
+    // Add code to be executed when the alert element is closed.
```

Comment:
This example should demonstrate using event.preventDefault() to keep the alert from closing

https://github.com/RedHat-UX/red-hat-design-system/blob/221a585ba610620953f940f2e72e66ba6314d006/elements/rh-alert/rh-alert.ts#L99

An example could be running some async function and only closing the alert when it resolves

Reply:
Perhaps this could be a second example in addition to the basic one of adding code after the close event is fired?  I was thinking having a basic one would be helpful for those not accustomed yet to adding event listeners to elements.